canonical/ubuntu.com
# WD-4481 Update the third party and geolocation legal document

## Done

-  Update the new third party and geolocation legal document

## QA

- Open the demo and go to /legal/data-privacy and check the document loads correctly 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4481
